File: Beecrowd/Python/bee1131.py
gremio = 0
inter = 0
tie = 0
grenais = 1

while 1:
    intGoals, greGoals = map(int, input().split())

    if intGoals > greGoals:
        inter += 1
    elif greGoals > intGoals:
        gremio += 1
    else:
        tie += 1

    print("Novo grenal (1-sim 2-nao)")
    continues = input()

    match continues:
        case "1":
            grenais += 1
        case "2":
            break

print(f"{grenais} grenais\nInter:{inter}\nGremio:{gremio}\nEmpates:{tie}")

if inter > gremio:
    print("Inter venceu mais")
elif gremio > inter:
    print("Gremio venceu mais")
else:
    print("Não houve vencedor")

File: Beecrowd/C/bee1131.c
#include <stdio.h>

int main() {
  int inter, gremio, newGrenal;
  int numberOfGrenais = 0, victoriesOfInter = 0, victoriesOfGremio = 0,
      nDraws = 0;

  while (1) {
    scanf("%d %d", &inter, &gremio);

    numberOfGrenais++;

    if (inter > gremio)
      victoriesOfInter++;
    else if (gremio > inter)
      victoriesOfGremio++;
    else
      nDraws++;

    printf("Novo grenal (1-sim 2-nao)\n");
    scanf("%d", &newGrenal);

    if (newGrenal == 2)
      break;
  }

  printf("%d grenais\n", numberOfGrenais);
  printf("Inter:%d\n", victoriesOfInter);
  printf("Gremio:%d\n", victoriesOfGremio);
  printf("Empates:%d\n", nDraws);

  if (victoriesOfInter > victoriesOfGremio)
    printf("Inter venceu mais\n");
  else if (victoriesOfGremio > victoriesOfInter)
    printf("Gremio venceu mais\n");
  else
    printf("Não houve vencedor\n");
}
